src/modules/auth.js

```diff
@@ -33,6 +32,32 @@ const login = async (event, dispatch, history) => {
   }
 };
 
+const signUp = async (event, dispatch, history) => {
+  event.preventDefault();
+  try {
+    const email = event.target.email.value;
+    const password = event.target.password.value;
```

Comment:
password confirmation?

Reply:
for what reason? we dont wanna save the password confirmation if it differs from password, and if it doesnt differs than we saves it into password